appengine-java21/bigtable/build.gradle

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 Google Inc.
+// Copyright 2024 Google LLC
```

Comment:
leave original copyright year

Reply:
Wait! no, I changed all of them with different years as well!
This is not google3 critique to go back to old versions or do inline changes in a nice cider editor!